google/cloud/bigtable/examples/bigtable_samples.cc

```diff
@@ -40,7 +40,7 @@ void RunTableOperations(google::cloud::bigtable::TableAdmin admin,
 
   std::cout << "Listing tables:\n";
   auto tables =
-      admin.ListTables(google::bigtable::admin::v2::Table::VIEW_UNSPECIFIED);
+      admin.ListTables(google::cloud::bigtable::TableAdmin::NAME_ONLY);
```

Comment:
Mention the value changes in the PR description?

Reply:
Done.